deploy/operator.yaml

```diff
@@ -25,6 +25,8 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: metadata.namespace
+            - name: APP_NAMESPACES
+                value: "unifiedpush"
```

Comment:
Hmm, this doesn't look like it does what it says in the commit message :wink: If we put the mdc namespace name (I'm not sure what it is) in here by default, would things break if it's not there?

Reply:
It doesn't need to be here at all, you're right, I got confused by some samples.